cmake/Modules/FindCurl_EP.cmake

```diff
@@ -163,8 +161,8 @@ if (NOT CURL_FOUND AND TILEDB_SUPERBUILD)
 
     ExternalProject_Add(ep_curl
       PREFIX "externals"
-      URL "https://curl.se/download/curl-7.74.0.tar.gz"
-      URL_HASH SHA1=cd7239cf9223b39ade86a14eb37fe68f5656eae9
+      URL "https://curl.se/download/curl-7.88.1.tar.gz"
+      URL_HASH SHA1=6ae5229c36badb822641bb14958e7d227c57611d
       CONFIGURE_COMMAND
         ${CMAKE_COMMAND} -E env PKG_CONFIG_PATH=${SSL_PKG_CONFIG_PATH} ${TILEDB_EP_BASE}/src/ep_curl/configure
```

Comment:
Can you move it to CMake?

Reply:
Move what to cmake? The build for each platform? Currently we use cmake on windows where it is required. We use autotools on linux/macos because historically its been more robustly supported by curl. If the cmake support is better now we could consider changing but that needs to be in a separate PR. We need to do a minimum version bump here to unblock the currently broken superbuild when building azure+curl. Switching to cmake involves a higher degree of risk than is acceptable for a backport for this defect fix.

Reply:
Yes I agree, the purpose of this PR is just a version bump.